x-pack/plugins/security_solution/common/api/detection_engine/rule_management/bulk_crud/bulk_delete_rules/bulk_delete_rules_route.gen.ts

```diff
@@ -30,3 +30,15 @@ export type BulkDeleteRulesRequestBodyInput = z.input<typeof BulkDeleteRulesRequ
 
 export type BulkDeleteRulesResponse = z.infer<typeof BulkDeleteRulesResponse>;
 export const BulkDeleteRulesResponse = BulkCrudRulesResponse;
+
+export type BulkDeleteRulesPostRequestBody = z.infer<typeof BulkDeleteRulesPostRequestBody>;
```

Comment:
We should use the `DELETE` and `POST` request and response types in the API route handler so the compiler will help us identify errors. Right now these appear to be unused.

Reply:
I added generated TS types as union to avoid code duplication.